zentral/contrib/osquery/serializers.py

```diff
@@ -220,3 +221,45 @@ def update(self, instance, validated_data):
         if compliance_check_deleted:
             query.refresh_from_db()
         return query
+
+
+class PackQuerySerializer(serializers.ModelSerializer):
+    pack = serializers.PrimaryKeyRelatedField(queryset=Pack.objects.all())
+    query = serializers.PrimaryKeyRelatedField(queryset=Query.objects.all())
+
+    class Meta:
+        model = PackQuery
+        fields = "__all__"
+
+    def validate_query(self, value):
+        slug = slugify(value.name)
```

Comment:
Can you confirm that the framework doesn't already catch this error?

Reply:
yes, If i dont catch this I get db error:
```
django.db.utils.IntegrityError: duplicate key value violates unique constraint "osquery_packquery_pack_id_slug_94e4132b_uniq"
DETAIL:  Key (pack_id, slug)=(8763, 0a8oqwevsh4r) already exists.
```